lemur/tests/test_oauth.py

```diff
@@ -0,0 +1,22 @@
+from lemur.auth.views import *  # noqa
+
+
+def test_build_hmac(client):
+    from lemur.auth.views import build_hmac
+
+    assert isinstance(build_hmac(), hmac.HMAC)
+
+
+def test_generate_state_token(client):
+    from lemur.auth.views import generate_state_token
+
+    assert generate_state_token()
+
+
+def test_verify_state_token(client):
+    from lemur.auth.views import generate_state_token
+    from lemur.auth.views import verify_state_token
+
+    token = generate_state_token()
+    assert verify_state_token(token)
+    assert not verify_state_token('123456:f4k8')
```

Comment:
We could add a test for staleness:

```python
token = generate_state_token()
assert verify_state_token(token)
time.sleep(15)
assert not verify_state_token(token)
```

But that would slow down every build by 15 seconds - is it worth including? (I don't think it's possible to patch a local variable in a function with `mock`)

Reply:
Are you able to use `freeze_time`? Freeze in past, generate token, unfreeze, verify stale? 
Sample: https://github.com/Netflix/lemur/blob/master/lemur/tests/test_certificates.py#L203
I haven't used it extensively so perhaps not an option, but I wanted to check if you tried that already.

Reply:
what about setting the tolerance to 0, would do that the trick?
`OAUTH_STATE_TOKEN_STALE_TOLERANCE_SECONDS = 0`

Reply:
Thank you both! I used a little bit from both suggestions: I define `OAUTH_STATE_TOKEN_STALE_TOLERANCE_SECONDS` in `lemur.tests.conf` and import it in my test, then apply `freeze_time` to grab a token that's at least the tolerance seconds stale. By pulling this from test config explicitly, we can be confident the the verification will also pull from our test tolerance seconds.